test_data/std/lookup_via_challenges.asm

```diff
@@ -1,25 +1,24 @@
-use std::prelude::Query;
```

Comment:
@georgwiese we don't seem to be testing this?

Reply:
That is true, see my comment below.